.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,32 @@
-### Pull Request check-list
+<!--
+Thank you for submitting a pull request to Node.js. Before you submit, please
+review below requirements and walk through the checklist.
 
-_Please make sure to review and check all of these items:_
+You can run the test suite by invoking: `make -j4 lint test` on linux or
+`vcbuild test nosign` on Windows.
 
-- [ ] Does `make -j8 test` (UNIX) or `vcbuild test nosign` (Windows) pass with
-  this change (including linting)?
-- [ ] Is the commit message formatted according to [CONTRIBUTING.md][0]?
-- [ ] If this change fixes a bug (or a performance problem), is a regression
-  test (or a benchmark) included?
-- [ ] Is a documentation update included (if this change modifies
-  existing APIs, or introduces new ones)?
+If this aims to fix a regression or you’re adding a feature, make sure you also
+write a test. Finally – if possible – a benchmark that quantifies your changes.
 
-_NOTE: these things are not required to open a PR and can be done
-afterwards / while the PR is open._
+Finally, read through our contributors guide and make adjustments as necessary:
+https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
+-->
 
-### Affected core subsystem(s)
+##### Checklist
 
-_Please provide affected core subsystem(s) (like buffer, cluster, crypto, etc)_
+<!-- remove lines that does not apply to you -->
 
-[0]: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#step-3-commit
+- tests and code linting passes
+- a test and/or benchmark is included
+- documentation is changed or added
+- the commit message follows commit guidelines
```

Comment:
hmmm i was not under that impression

My comment was about making sure these things were actually a checklist (i.e. tests pass).